src/utils/lookup-table-utils.ts

```diff
@@ -54,11 +56,16 @@ export const languageMatcherCaseStatement = (languageColumn: string | undefined)
   SUPPORTED_LOCALES.map((locale) => {
     const lang = locale.split('-')[0].toLowerCase();
     const tLang = lang;
-    languageMatcher.push(`WHEN LOWER("${languageColumn}") LIKE '${lang}%' THEN '${locale.toLowerCase()}'`);
+    languageMatcher.push(pgformat('WHEN LOWER(%I) LIKE %L THEN %L', languageColumn, `%${lang}%`, locale.toLowerCase()));
     SUPPORTED_LOCALES.map((locale) => {
       const lang = locale.split('-')[0].toLowerCase();
       languageMatcher.push(
-        `WHEN LOWER("${languageColumn}") LIKE '%${t(`language.${lang}`, { lng: tLang }).toLowerCase()}%' THEN '${locale.toLowerCase()}'`
+        pgformat(
+          'WHEN LOWER(%I) LIKE %L THEN %L',
```

Comment:
Same q as above - repeated placeholder.

Reply:
`%L` is a SQL Litteral/Value and` %I` is a SQL Identifier and `%s` is a raw string. The library replaces things in the order it's given. It works in much the same was `printf`

Reply:
Old comment... I removed validation a little while ago... Working on the assumption that to get to this stage we've already validated the lookup table.  So I'll remove the comment